test/main.test.ts

```diff
@@ -541,6 +581,48 @@ describe("Payloads", () => {
     );
   });
 
+  test.skip("a Pull Request with 1 new SVG and one new SVG in a directory", async () => {
```

Comment:
Test should work just fine now, no reason to be `skip`ped anymore :slightly_smiling_face: 

```suggestion
  test("a Pull Request with 1 new SVG and one new SVG in a directory", async () => {
```